### ISSUE TITLE
Warpinator-send fail but warpinator gui works

### ISSUE THREAD

**Issue by DarkWizarD24** [linuxmint/warpinator#197]
### Distribution

Mint 21.2

### Package version

1.6.4

### Frequency

Always

### Bug description

I can send file using the GUI but not warpinator-send.

When I send a file using warpinator-send I have an error popup and this is shown in the GUI

![image](https://github.com/linuxmint/warpinator/assets/8056086/f0c25d6e-
f60b-490d-9f65-276d093c613b)

Log when I send using warpinator-send
[warpinator output.txt](https://github.com/linuxmint/warpinator/files/13423636/warpinator.output.txt)

Log when I send using warpinator gui
[warpinator success output.txt](https://github.com/linuxmint/warpinator/files/13423669/warpinator.success.output.txt)

### Steps to reproduce

```shell
warpinator-send /home/<my user>/Bureau/test.py
```

### Expected behavior

The file is sent

**Comment by mtwebster** [comment]
I think this may be because warpinator-send only accepts uris, not simple paths. We fixed this recently in 17156747b071a2

You can confirm it by using a uri format:
```
warpinator-send file:///home/<my user>/Bureau/test.py
```

**Comment by DarkWizarD24** [comment]
It's works. Thanks you
